perf.groovy

```diff
@@ -27,8 +27,22 @@ def static getOSGroup(def os) {
 // Setup perflab tests runs
 [true, false].each { isPR ->
     ['Windows_NT'].each { os ->
-		['x64', 'x86'].each { architecture ->
-			def newJob = job(Utilities.getFullJobName(project, "perf_perflab_${os}_${architecture}", isPR)) {
+		['x64', 'x86ryujit', 'jit32'].each { arch ->
+            architecture = arch
+            testEnv = ''
+
+            if (arch == 'jit32')
+            {
+                architecture = 'x86'
+                testEnv = '-testEnv %WORKSPACE%\\tests\\x86\\compatjit_x86_testenv.cmd'
+            }
+            else if (arch == 'x86ryujit')
```

Comment:
I think we can get rid of this entire branch.  The ryujit_x86 command that we call does not actually do anything so it does not seem like we need to call it.